src/routes/board.py

```
@@ -37,7 +37,7 @@ async def create_board(session: SessionDep, request: Request, body: BodyCreateBo
     except IntegrityError:
         session.rollback()
         logger.warning(f'err_type=board_update err_code=409 ; msg=unique field already exists ; executor={current_user.id}')
-        raise HTTPException(status_code=409, detail="unique field already exists")
+        raise HTTPException(tatus_code=409, detail="unique field already exists")
```

Ruff (0.13.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)